src/lib/docs/ContractKit/GetMethod.md

```diff
@@ -1,9 +1,35 @@
 ---
 title: Get (Method)
-description: change_me
+description: A method to retrieve specific rows from a blockchain table based on certain query parameters.
 category: ContractKit
 published: true
 order: 5
 ---
 
-How to use the `table.get` method to pull data
+# Get
+
+The `get` method allows for precise retrieval of rows from a [table](/docs/contract-kit/table). By specifying query parameters, users can pinpoint a particular row or a set of rows they want to fetch.
+
+## Usage
+
+```typescript
+const row = await contract.table('table_name').get(index_value, queryParams);  
```

Comment:
Like I mentioned in the other comment - start with the most simple example:

```
table.get()
```

Then write other examples that explain `get(value)` and `get(value, queryParams)` to describe the multiple ways this can be used.